core/osrd-railjson/src/main/java/fr/sncf/osrd/railjson/schema/rollingstock/RJSEtcsBrakeParams.java

```diff
@@ -0,0 +1,81 @@
+package fr.sncf.osrd.railjson.schema.rollingstock;
+
+import com.squareup.moshi.Json;
+
+public class RJSEtcsBrakeParams {
+
+    // A_brake_emergency: the emergency deceleration curve (values in m/s²)
```

Comment:
We could explain where to look for these names (`A_brake_emergency` and such).

Ideally we should directly link the PDF somewhere, even if it's not publicly available (I think?)